docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -0,0 +1,222 @@
+
+```yaml
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-pelorus-instance
+  namespace: pelorus
+spec:
+  exporters:
+    [...] # Pelorus exporters configuration options
+  # Pelorus Core configuration options
+```
+
+For configuration options for exporters, check [its configuration guide](PelorusExporters.md).
+
+## Example
+
+Configuration part of the Pelorus object YAML file, with some non-default options:
+
+```yaml
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-instance
+  namespace: pelorus
+spec:
+  exporters:
+    [...] # Pelorus exporters configuration options
+  # Below are non-default options
+  openshift_prometheus_basic_auth_pass: mysecretpassword
+  openshift_prometheus_htpasswd_auth: 'internal:{SHA}CM2SM2eJAAllfquBJ1M3m9syHus='
+  prometheus_retention: 500d
+  prometheus_retention_size: 2GB
+  prometheus_storage: true
+  prometheus_storage_pvc_capacity: 3Gi
+  prometheus_storage_pvc_storageclass: mystorageclass
+```
+
+## List of all configuration options
+
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [[exporters]](PelorusExporters.md) | yes | - |
+| [prometheus_retention_size](#prometheus_retention_size) | no | `1GB` |
+| [prometheus_retention](#prometheus_retention) | no | `1y` |
+| [prometheus_storage](#promtheus-storage) | no | `false` |
+| [prometheus_storage_pvc_capacity](#prometheus_storage_pvc_capacity) | no | `2Gi` |
+| [prometheus_storage_pvc_storageclass](#prometheus_storage_pvc_storageclass) | no | `gp2` |
+| [openshift_prometheus_htpasswd_auth](#openshift_prometheus_htpasswd_auth)  | no | `internal:{SHA}+pvrmeQCmtWmYVOZ57uuITVghrM=` |
+| [openshift_prometheus_basic_auth_pass](#openshift_prometheus_basic_auth_pass) | no | `changeme` |
+| [[extra_prometheus_hosts]](#extra_prometheus_hosts) | no | - |
+| [thanos_version](#thanos_version) | no | `v0.28.0` |
+| [bucket_access_point](#bucket_access_point) | no | - |
+| [bucket_access_key](#bucket_access_key) | no | - |
+| [bucket_secret_access_key](#bucket_secret_access_key) | no | - |
+| [thanos_bucket_name](#thanos_bucket_name) | no | `thanos` |
+| [custom_ca](#custom-pki) | no  | - |
+
+## Prometheus
+
+Pelorus allows to configure few aspects of [Prometheus](https://prometheus.io/), that is deployed as an [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) available from the [OLM dependency](https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/) mechanism.
+
+For detailed information about planning Prometheus storage capacity and configuration options please refer to the [operational aspects](https://prometheus.io/docs/prometheus/latest/storage/#operational-aspects) of the Prometheus documentation.
+
+### Prometheus Data Retention
+
+###### prometheus_retention_size
+
+- **Required:** no
```

Comment:
could we add type as well? And since default value is not always present, I would put it as second level. Example

- **Required:** no
    - **Default Value:** 1GB
- **Type:** string